### PR TITLE
Update seg_to_regions.py

### PR DESCRIPTION
**Pull request by mansourhas**
fix error message ` isin() received an invalid combination of arguments - got (Tensor, tuple), `

### CODE REVIEW

**Comment by mansourhas** [comment]
this issue happens when I'm using CPU, but now, I'm running from a different environment and got a warning..

` UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor).
  region_output[:, region_id] |= torch.isin(segmentation[:, self.channel_in_seg], torch.tensor(region_labels).to(segmentation.dtype))`

which might suggest the issue that the `region_labels` is a tuble only happens when using CPU?

**Comment by FabianIsensee** [comment]
The regions are supposed to be nested Tuples / Lists of int. In your case they seem to be torch.Tensor already which gives rise to this warning. See the type hints (which I just updated to make this more explicit)
